lib/roken/roken.h.in

```diff
@@ -1260,4 +1260,20 @@ int ROKEN_LIB_FUNCTION rk_socket(int, int, int);
 #include <socket_wrapper.h>
 #endif
 
+#ifndef LLONG_MAX
+#ifdef LONG_LONG_MAX
+#define LLONG_MAX LONG_LONG_MAX
+#endif
+#endif
+
+#ifndef ULLONG_MAX
+#ifdef ULONG_LONG_MAX
+#define ULLONG_MAX ULONG_LONG_MAX
+#endif
+#endif
+
+#ifndef LLONG_MIN
+#define LLONG_MIN (-LLONG_MAX - 1LL)
```

Comment:
Just so we're clear, I'm perfectly happy to assume twos' complement even though the C standards don't require it.  Anyone who runs into a platform that uses ones' complement or whatever can fix it when they do.